docs/admin/kubernetes-ops/logs.md

```diff
@@ -3,17 +3,318 @@ tags:
+    appLoggers:
+      configApiLogLevel: "TRACE"
+      persistenceLogLevel: "TRACE"
+      persistenceDurationLogLevel: "TRACE"
+      ldapStatsLogLevel: "TRACE"
+      scriptLogLevel: "TRACE"
+............
+............      
+```
+
+Apply the changes: 
+
+```bash
+helm upgrade <helm-release-name> janssen/janssen -f override.yaml -n <namespace>
+```
+
+View the logs of config-api:
+```bash
+kubectl logs <config-api-pod> -n <namespace>
+```
+
+
+### fido2
+To get the current log level of fido2:
+
+```bash
+kubectl get configmap -n <namespace> janssen-config-cm -o yaml | grep CN_FIDO2_APP_LOGGERS 
+```
+
+Example output:
+```yaml
+CN_FIDO2_APP_LOGGERS: 
+'{"
+fido2_log_level":"INFO",
+"fido2_log_target":"STDOUT",
+"persistence_log_level":"INFO",
+"persistence_log_target":"FILE"}'
+```
+
+To override the current logging level in fido2, you can either change it directly using `kubectl edit` command, or add the desired changed to a yaml file and apply it using `helm`: 
+
+- Hack it - Change it directly:
+
+> **Warning**
+> This can cause the deployments to break, but if you wish you may edit it directly and restart the wanted deployment
+
+
+Edit using the following command: 
+```bash
+kubectl edit configmap <helm-release-name>-config-cm -n <namespace>
+```
+
+Restart the wanted deployment:
+```bash
+kubectl rollout restart deployment <fido2-deployment> -n <namespace>
+```
+
+- Add changes to yaml:
+
+add these changes to your `override.yaml` file:
+
+```yaml
+............
+............
+global:
+  fido2:
+    appLoggers:
+      fido2LogLevel: "TRACE"
+      persistenceLogLevel: "TRACE"
+............
+............      
+```
+
+Apply the changes: 
+
+```bash
+helm upgrade <helm-release-name> janssen/janssen -f override.yaml -n <namespace>
+```
+
+View the logs of fido2:
+```bash
+kubectl logs <fido2-pod> -n <namespace>
+```
+
+
+### scim
+To get the current log level of scim:
+
+```bash
+kubectl get configmap -n <namespace> janssen-config-cm -o yaml | grep CN_SCIM_APP_LOGGERS 
+```
+
+Example output:
+```yaml
+CN_SCIM_APP_LOGGERS: 
+'{"
+ldap_stats_log_level":"INFO",
+"ldap_stats_log_target":"FILE",
+"persistence_duration_log_level":"INFO",
+"persistence_duration_log_target":"FILE",
+"persistence_log_level":"INFO",
+"persistence_log_target":"FILE",
+"scim_log_level":"INFO",
+"scim_log_target":"STDOUT",
+"script_log_level":"INFO",
+"script_log_target":"FILE"}'
+```
+
+To override the current logging level in scim, you can either change it directly using `kubectl edit` command, or add the desired changed to a yaml file and apply it using `helm`: 
+
+- Hack it - Change it directly:
+
+> **Warning**
+> This can cause the deployments to break, but if you wish you may edit it directly and restart the wanted deployment
+
+
+Edit using the following command: 
+```bash
+kubectl edit configmap <helm-release-name>-config-cm -n <namespace>
+```
+
+Restart the wanted deployment:
+```bash
+kubectl rollout restart deployment <scim-deployment> -n <namespace>
+```
```

Comment:
Move this section to the bottom of the page and remove it from all other places. Place a title on it `#### Hack it - change it directly` then in the above lines where you write `To override the current logging level in scim, you can either change it directly using `kubectl edit` command, or add the desired changed to a yaml file and apply it using `helm`:` link the `kubectl edit ` to that section. Also put helm instructions first then `or you can either change it directly using `[kubectl edit](#link to hack here) command`